server/routerlicious/packages/routerlicious-base/src/riddler/tenantManager.ts

```diff
@@ -236,6 +240,18 @@ export class TenantManager {
      * Retrieves the secret for the given tenant
      */
     public async getTenantKeys(tenantId: string, includeDisabledTenant = false): Promise<ITenantKeys> {
+        const lumberProperties = { [BaseTelemetryProperties.tenantId]: tenantId };
+
+        // Read from cache first
+        const cacheId = `tenantKeys:${tenantId}`;
```

Comment:
nit: typical pattern for this is to add a `private getCacheKey(tenantId: string): string;` method to the class, and use that when getting a key for reading/writing to cache. Eliminates possibility for a typo

Reply:
Created additional functions to help with cache operations